### DISCUSSION TITLE
What models should be run?

### DISCUSSION THREAD

**Discussion by KennethEnevoldsen** [embeddings-benchmark/mteb#286] [General]
This poll is to get an idea about what models should be run. The hope is to create a minimal

### Initial list:

- multilingual-e5-large-instruct
- multilingual-e5-{size}
- paraphrase-multilingual-mpnet-base-v2
- Commercial APIs (if sponsored)
  - text-embedding-3-{size}
  - embed-multilingual-v3.0

@MartinBernstorff, @imenelydiaker, @Muennighoff

**Comment by imenelydiaker** [comment]
I think `text-embedding-3-large` from OpenAI is expensive to run (time and money). The small would be fine.
But who will run the commercial API's models? If people already have API access it's okay, but I don't expect us to pass API keys to people, unless it's temporary..

Do we have a budget or something?
